arviz/data/io_pyro.py

```diff
@@ -183,13 +211,25 @@ def to_inference_data(self):
                 "sample_stats": self.sample_stats_to_xarray(),
                 "log_likelihood": self.log_likelihood_to_xarray(),
                 "posterior_predictive": self.posterior_predictive_to_xarray(),
+                "predictions": self.predictions_to_xarray(),
                 **self.priors_to_xarray(),
                 "observed_data": self.observed_data_to_xarray(),
             }
         )
 
 
-def from_pyro(posterior=None, *, prior=None, posterior_predictive=None, coords=None, dims=None):
+def from_pyro(
+    posterior=None,
+    *,
+    prior=None,
+    posterior_predictive=None,
+    predictions=None,
+    coords=None,
+    dims=None,
+    pred_dims=None,
+    idata_origin=None,
+    inplace=False
```

Comment:
I would not add this to `from_pyro`, they could go to a `from_pyro_predictions` (to mimic pymc3 pattern) but I am not sure it is worth it to include such a function it yet, it would basically be only a `concat` call, so users can call `az.concat` directly